### ISSUE TITLE
return ERROR on some translation

### ISSUE THREAD

**Issue by ajlucero** [lushan88a/google_trans_new#5]
Hi ! Thank you for your great work, I can't deny the fact that this is so useful on the project I am trying on :) I highly recommend this work of yours.

I just have encounter this return ERROR string, I hope to resolve or pin point what might be wrong while translating the text

I'm actually running 38K+ translation in 4 threads'

text: 這是……<size=24>欸欸欸——？？</size>\n難道你想和我<color=#CA2E0AFF>在噴泉前留下紀念照</color>嗎？這主意太棒了，如此就能帶走最喜愛的風景了！我先去噴泉前面等你~快，快！

result : ERROR

**Comment by lushan88a** [comment]
Thanks for you question.
I try use your text . But i don't get error.
``` python
t = google_translator(timeout=10)
text = "這是……<size=24>欸欸欸——？？\n難道你想和我<color=#CA2E0AFF>在噴泉前留下紀念照嗎？這主意太棒了，如此就能帶走最喜愛的風景了！我先去噴泉前面等你~快，快！"
print(text)
text_trans = t.translate(text,lang_src='zh',lang_tgt='en')
print(text_trans)
```
-> This is... <size=24> Hey Hey--? ? Do you want to <color=#CA2E0AFF> with me to leave a souvenir photo in front of the fountain? This is a great idea, so you can take away your favorite scenery! I'll wait for you in front of the fountain~ Quick, quick! 

Can you tell the detail for your error?

**Comment by ajlucero** [comment]
Hi the error I think only show via file reader.

I read a Chinese text file by line and put it into threading
most of the lines have \n (newline escape sequence) which cause the translation to return string "ERROR"

hope this helps

I actually try that scenario you did, it wont trigger any error cause it only happened when it is read via file

**Comment by hockeymikey** [comment]
I'm getting this error return too. Seems to happen on some non-existent language codes I use or it seems if I'm using a single world like say "Google" that doesn't translate or even if it does returns "ERROR". That's all I sadly have, no additional info.

**Comment by hockeymikey** [comment]
Expecting ',' delimiter: line 1 column 41 (char 40) is being thrown on the first json.loads of that section. Instead of error you should be returning the exception instead.

**Comment by hockeymikey** [comment]
@lushan88a Yep, I just installed it like yesterday, so it's 1.1.1. My code is exactly as you posted. Nothing different, and the lang codes are valid. So debugging '[["wrb.fr","MkEWBc",null,null,null,[13],]' is constructed and called on that first json that it errors on. If you add another bracket at the end and then you escape out the double quotes it works. I can't figure out to replace those double quotes because it turns them into double backslash double quotes, but testing with a static string with it manually escaped out proved working in my tests.
